semantic/monotype.go

```diff
@@ -160,11 +160,12 @@ func monoTypeFromVar(v *fbsemantic.Var) MonoType {
 }
 
 func getFun(tbl fbTabler) (*fbsemantic.Fun, error) {
-	f, ok := tbl.(*fbsemantic.Fun)
-	if !ok {
-		return nil, errors.New(codes.Internal, "MonoType is not a function")
-	}
+	f := tbl.(*fbsemantic.Fun)
 	return f, nil
+	// if !ok {
+	// 	return nil, errors.New(codes.Internal, "MonoType is not a function")
+	// }
+	// return f, nil
```

Comment:
If this is no longer needed can you remove it?

Reply:
I removed this so I could force a panic and find the issue. In the future, I think it would be good to force that panic when using this incorrectly, but I readded it because that isn't the present.